code/script.js

```diff
@@ -1,51 +1,215 @@
+  } else if (lineNumber === 7) {
+    userReply(message)
+    userInput.value = ''    
+    setTimeout(() => eighthLine (message), 1000) ; 
+  } else if (lineNumber === 8) {
+    userReply(message)
+    userInput.value = ''    
+    setTimeout(() => ninthLine (message), 1000) ; 
+  } else {
+    userReply (message)
+    userInput.value=""
+    setTimeout(() => tenthLine (message), 1000) ; 
+
+}; 
+}
+
 // Starts here
-const greetUser = () => {
-  // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot');
-  // Just to check it out, change 'bot' to 'user' here 👆
+
+
+//First reply from bot 
+const firstLine = () => {
+  lineNumber = 1; 
+  botReply(`Knock, knock.`);
 }
 
+//Second reply from bot
+const secondLine = () => {
+lineNumber++;  
+botReply(`Tank.`);
+} 
+
+//Third reply from bot
+  const thirdLine = () => {
+  lineNumber++
+  botReply(`You're welcome!`);
+  setTimeout(() => botReply (`Knock, knock.`), 2000) ; 
+  }
+
+  //Fourth reply from bot 
+  const fourthLine = () => {
+    lineNumber++;  
+    botReply(`Boo.`);
+  }
+  
+  //Fitfh reply from bot 
+  const fifthLine = () => {
+    lineNumber++;  
+    botReply(`Don't cry!`);
+    setTimeout(() => botReply (`Knock, knock.`), 2000) ; 
+  }
+
+  //Sixth reply from bot 
+    const sixthLine = () => {
+    lineNumber++;  
+    botReply(`Ash.`);
+  }
+  
+  //Seventh reply from bot 
+  const seventhLine = () => {
+    lineNumber++;  
+    botReply(`Sounds like you have a cold.`);
+    setTimeout(() => botReply (`Knock, knock.`), 2000) ; 
+  }
+  
+  //Eighth reply from bot 
+  const eighthLine = () => {
+    lineNumber++;  
+    botReply(`Who.`);
+  }
+
+  //Ninth reply from bot 
+  const ninthLine = () => {
+    lineNumber++;  
+    botReply(`What are you? An owl?`);
+    setTimeout(() => botReply (`That's all I have for today.`), 2000) ; 
+    setTimeout(() => botReply (`I hope you enjoyed it!`), 3000) ;
+  }
+
+  //Tenth reply from bot that I'll use if I have more time to work on the bot. 
+  const tenthLine = () => {
+    lineNumber++;  
+    botReply(`Say whaaaat?`);
+  }
+
+// Here is the function for the user's input 
+
 // Set up your eventlisteners here
+  //Initial button click from user, here the user enters their question. If someone hits enter it works too.   
+userForm.addEventListener('submit', (event) => {
+event.preventDefault();
+
+//Store the value in a variable so I can access it after we clear it from the input. 
+
+let userText = userInput.value ; 
+userInput.value = '' //Clears input field after submitting
+
+})
+
+//Makes the chat bot move on to the next line.
+sendBtn.addEventListener ('click', () => nextLine(userInput.value)) 
+
+userInput.addEventListener('keypress', (event) => {
+  if (event.key === 'submit' && userInput.value) nextLine(userInput.value)
+  }); 
```

Comment:
Could this section have been solved with less addEventListener? I don't know the answer to this, but I'm thinking that since the user input will always be in the same text field and submitted the same way. Maybe there is a simpler way write it.